pkg/clusterapi/system.go

```diff
@@ -172,7 +172,20 @@ func (c *system) Run(ctx context.Context, installConfig *installconfig.InstallCo
 			),
 		)
 	case gcp.Name:
-		// TODO
+		controllers = append(controllers,
+			c.getInfrastructureController(
+				&GCP,
+				[]string{
+					"-v=2",
+					"--metrics-bind-addr=0",
+					"--health-addr={{suggestHealthHostPort}}",
+					"--webhook-port={{.WebhookPort}}",
+					"--webhook-cert-dir={{.WebhookCertDir}}",
+					"--feature-gates=BootstrapFormatIgnition=true,ExternalResourceGC=true",
```

Comment:
These can be removed for now?

Reply:
@vincepri is this because ignition is not currently working for GCP yet? I can definitely comment it out, but eventually this will have to be put back.